securedrop/config_tool.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import re
+import sys
+import yaml
+
+from os import path
+from yaml.scanner import ScannerError
+
+
+"""
+A CLI tool to help admins configure their SecureDrop instances. Uses argparse, so run this file from
+your terminal with --help to get instructions on how to use this.
+"""
+
+
+def init_logger():
+    l = logging.getLogger(__file__)
+    l.setLevel(logging.DEBUG)
+    ch = logging.StreamHandler(sys.stdout)
+    ch.setLevel(logging.DEBUG)
+    ch.setFormatter(logging.Formatter('%(levelname)s - %(message)s'))
+    l.addHandler(ch)
+    return l
+
+log = init_logger()
+
+
+class ConfigToolException(Exception):
+
+    def __init__(self, msg=None):
+        super(ConfigToolException, self).__init__(msg)
+
+
+def check_ip_address_valid(ip):  # "good enough"
+    m = re.match(r"^(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})$", ip)
+    if bool(m) and all(map(lambda n: 0 <= int(n) <= 255, m.groups())):
```

Comment:
You don't have to use `bool` here. `""`, `0`, `set()`, `None` and the trivial instance/ zero object of most of the builtin types is equivalent to `False` when `if` testing a condition.